ontology/yaml/resources/HVAC/entity_types/DOAS.yaml

```diff
@@ -151,3 +151,21 @@ DOAS_CHWDC_EFSS_EFVSC_EFDPM_PHRUD_HWDC_DFSS_DFVSC_DSPM_DFDPM:
   - DSPM
   - DFDPM
 
+DOAS_SFVSC_EFVSC_HTWHLSTC_CHWVM_HWVM_SDM_EDM_SFM_FDPSM_CO2C_SDM2X_EDM2X:
+  description: "Multi-zone chilled water and heating water DOAS with heat wheel, supply, exhaust fan speed control"
+  is_canonical: true
+  implements:
+  - AHU
+  - SFVSC
+  - EFVSC
+  - HTWHLSTC
+  - CHWVM
+  - HWVM
+  - SDM
+  - EDM
+  - SFM
+  - FDPSM
+  - CO2C
```

Comment:
How can you have a single CO2 zone control with a multi-zone unit?

Reply:
so this would be an average of the different co2 sensors, the control isn't different for the different zones